### PR TITLE
feat_TT-77_Refactor_Tour_Events

### PR DESCRIPTION
**Pull request by mzolovska**
…ng the records, also added some testing

**JIRA:** 🌸 TT-something 🌸
https://champlainsaintlambert.atlassian.net/jira/software/projects/TT/boards/201/backlog?selectedIssue=TT-77
## Context:
Needed to add hotels to the tour events.
✨ What's happening? Let's dive into the magical story! ✨

### backend:
🔧 Backend sorcery to make everything super smooth! ✨
Added hotel to tour event and added more data to the db

### front end:
🌸 Added extra cuteness to the UI and made it sparkle! 🌈
Added the hotels to the tour events and added the sorting so that tour events are well displayed.

## Before and After UI (Required for UI-impacting PRs):
🎀 Please attach screenshots so we can see the adorable transformation! 🐾
image

### CODE REVIEW

**Comment by ioannisp03** [comment]
When u can, just edit the PR name so it follows the naming convention we have established

**Review by CharlesS42 — COMMENTED** [review]
Please make sure that the column in the frontend grid is named properly for a user (not "hotelLabel")

File: fe-traveltrove/src/features/staff/components/TourEventsTab.tsx [hunk 1]
```diff
@@ -177,7 +177,7 @@ const TourEventsTab: React.FC<TourEventsTabProps> = ({ tourId }) => {
               <th>{t("seqLabel")}</th>
               <th>{t("descLabel")}</th>
               <th>{t("eventLabel")}</th>
-              <th>{t("hotelLabel")}</th>
```
Comment (CharlesS42) [inline fe-traveltrove/src/features/staff/components/TourEventsTab.tsx]: Thank you for changing it!

File: fe-traveltrove/src/features/staff/components/EventsTab.tsx [hunk 2]
```diff
@@ -235,7 +236,7 @@ const EventsTab: React.FC = () => {
               <thead className="bg-light">
                 <tr>
                   <th>{t("nameE")}</th>
-                  <th>{t("actionsE")}</th>
```
Comment (CharlesS42) [inline fe-traveltrove/src/features/staff/components/EventsTab.tsx:238]: Good catch!

**Review by CharlesS42 — APPROVED** [review]

**Pull request merged**